specs/slider/Spec.md

```diff
@@ -32,167 +43,177 @@ Sliders are great for adjusting settings that reflect intensity levels (volume,
+#### Validation
 
+| Case                          | Handling          |
+| ----------------------------- | ----------------- |
+| `value` is less then `min`    | set to `min`      |
+| `value` is greater than `max` | set to `max`      |
+| `value` is empty              | set to `min` OR 0 |
 
 
-## Examples
 
-#### Simple
+#### Edge case handling
 
-The `defaultValue` property sets the initial position of the slider. 
+| Case                                     | Handling                                 | Example                                  |
+| ---------------------------------------- | ---------------------------------------- | ---------------------------------------- |
+| value is out of `step`                   | keep current value & display handle at the position that represents this value | `min=0` `max=10` `step=2` `value=3.5`. Handle is displayed at the position that represents 3.5. When user changes value (keyboard / mouse) we move it to the next expected, which is 4 or 2 |
+| `step` is not set & `displayMarks` is set to true | show marks at the beginning and at the end of the slider |                                          |
 
-![simpleExample](./assets/simpleExample.png)
 
 
+### Input Methods 
 
-#### Stepped
+#### Keyboard
 
-By default, the slider is continuous. The step property causes the slider to move in discrete increments.
+| Keys                                 | Action                      |
+| ------------------------------------ | --------------------------- |
+| up / right arrow key                 | increase value              |
+| left / down arrow key                | decrease value              |
+| home ( fn/ctrl + left arrow key)     | set min value               |
+| end ( fn/ctrl + left arrow key)      | set min value               |
+| page up (fn/ctrl + up arrow key)     | increase value by X         |
+| page down (fn/ctrl + down arrow key) | decrease value by X         |
+| tab                                  | moves to next element       |
+| shift + tab                          | moves to previous element   |
+| enter                                | -                           |
+| esc                                  | removes focus (if in focus) |
 
+**RTL** orientation
 
+| Keys                   | Action         |
+| ---------------------- | -------------- |
+| up  / left arrow key   | increase value |
+| down / right arrow key | decrease value |
 
-![steppedExample](./assets/steppedExample.png)
 
-See W3 org slider for reference - https://www.w3.org/TR/wai-aria-practices/examples/slider/slider-2.html
 
+### Mouse
 
+| Event                                   | Action                                   |
+| --------------------------------------- | ---------------------------------------- |
+| hover (over bar / handle / slider area) | highlight slider (both bar & handle)     |
+| click & drag (right / up)               | increase value                           |
```

Comment:
> click & drag (right / up)               | increase value 

I think `click & drag                | change value according to direction of movement` would sound more truly because we have reversed axises too.